R/register_extendr.R

```diff
@@ -12,20 +12,45 @@
 #' @param path File path to the package for which to generate wrapper code.
 #' @param quiet Logical indicating whether any progress messages should be
 #'   generated or not.
+#' @param force_wrappers Logical indicating whether to generate a minimal
+#'   wrapper in the cases when the package's namespace cannot be loaded. This is
+#'   useful to recover the wrapper file when something is wrong with it.
 #' @return The generated wrapper code. Note that this is not normally needed,
 #' as the function saves the wrapper code to `R/extendr-wrappers.R`.
 #' @export
-register_extendr <- function(path = ".", quiet = FALSE) {
+register_extendr <- function(path = ".", quiet = FALSE, force_wrappers = FALSE) {
   x <- desc::desc(rprojroot::find_package_root_file("DESCRIPTION", path = path))
   pkg_name <- x$get("Package")
 
   if (!isTRUE(quiet)) {
     message(glue("Generating extendr wrapper functions for package: {pkg_name}"))
   }
 
+  entrypoint_c_file <- rprojroot::find_package_root_file("src", "entrypoint.c", path = ".")
+  if (!file.exists(entrypoint_c_file)) {
+    stop(
+      "src/entrypoint.c, which is necessary to make wrappers available to R sessions, doesn't exist.",
```

Comment:
Implemented the check on `src/entrypoint.c`. But I'm still thinking what the error message should be here...

Reply:
Maybe write: "Could not find `src/entrypoint.c`. Are you sure this package is using extendr Rust code?"

Reply:
Sounds good, thanks.